File: app/src/main/java/co/za/takeabyte/gradalot2020/deals/IDealsView.java
package co.za.takeabyte.gradalot2020.deals;

import java.util.List;

import co.za.takeabyte.gradalot2020.promotions.uimodel.UIModelPromotionItem;

public interface IDealsView {

    void configureDealsContainer();

    void renderContentItems(List<UIModelPromotionItem> items);

}

File: app/src/main/java/co/za/takeabyte/gradalot2020/deals/IDealsView.kt
package co.za.takeabyte.gradalot2020.deals

import co.za.takeabyte.gradalot2020.promotions.uimodel.UIModelPromotionItem

interface IDealsView {

    fun configureDealsContainer()

    fun renderContentItems(items: List<UIModelPromotionItem>)

}